tests/Microsoft.DotNet.UpgradeAssistant.TestHelpers/UpgradeStepExtensions.cs

```diff
@@ -18,5 +20,17 @@ public static void SetStatus(this UpgradeStep step, UpgradeStepStatus status)
 
             method.Invoke(step, new object[] { status });
         }
+
+        public static void ClearRazorDocuments(this RazorUpdaterStep step)
+        {
+            var field = typeof(RazorUpdaterStep).GetField("_razorDocuments", BindingFlags.NonPublic | BindingFlags.Instance);
```

Comment:
maybe makes sense to use a folder path already included in the .gitignore file
https://github.com/github/gitignore/blob/master/VisualStudio.gitignore

e.g. "/bld"

#293

Reply:
I'm afraid I'm not following. This method is just a helper to clear the internal store of processed Razor documents for test purposes (to make sure they're regenerated at the right time).

Reply:
I think I misunderstood. Are the razor documents written to disk at this point?

Reply:
The test razor files are present before the test runs. The tests I have so far just make sure they're read correctly. Nothing is changed yet (that's coming in the next PR :))